rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -47,6 +48,7 @@ private BookKeeperServer()
     public static void main(String[] args)
     {
         conf = new Configuration();
+        conf.addResource(new YarnConfiguration());
```

Comment:
Bookkeeper is common for both hadoop1 and hadoop2. Now sure how hadoop1 cluster will work with yarn configuration .